### ISSUE TITLE
Sound_EnableSFX variable

### ISSUE THREAD

**Issue by vampik** [davidmarshall15/Healbot#17]
HealBot corrupts Sound_EnableSFX variable. It looks like **SET Sound_EnableSFX "1;"** in config.wtf (note the semicolon). Fixed by removing excessive semicolons in HealBot_Action.lua.

```
--- HealBot_Action.lua.bak	Sat Aug 22 14:14:04 2015
+++ HealBot_Action.lua	Wed Sep 23 12:23:41 2015
@@ -3115,24 +3115,24 @@
         scText="/cast [@"..unit..","..spellType.."] "..spellName..";\n"
     end
 
-    if HealBot_Globals.MacroSuppressSound==1 and sysSoundSFX=="1" then smName=smName.."/console Sound_EnableSFX 0;\n" end
+    if HealBot_Globals.MacroSuppressSound==1 and sysSoundSFX=="1" then smName=smName.."/console Sound_EnableSFX 0\n" end
     if HealBot_Globals.MacroSuppressError==1 then smName=smName.."/script UIErrorsFrame:Hide();\n" end
     if spellTar then smName=smName.."/target "..unit..";\n" end
     if spellTrin1 then smName=smName.."/use 13;\n" end
     if spellTrin2 then smName=smName.."/use 14;\n" end
     if HealBot_Config.MacroUse10==1 then smName=smName.."/use 10;\n" end
     if HealBot_Globals.MacroSuppressError==1 then smName=smName.."/script UIErrorsFrame:Clear(); UIErrorsFrame:Show();\n" end
-    if HealBot_Globals.MacroSuppressSound==1 and sysSoundSFX=="1" then smName=smName.."/console Sound_EnableSFX 1;\n" end
+    if HealBot_Globals.MacroSuppressSound==1 and sysSoundSFX=="1" then smName=smName.."/console Sound_EnableSFX 1\n" end
     smName=smName..scText
     if spellAvoidBC then smName=smName.."/use 4;" end
     if strlen(smName)>255 then
         smName=""
-        if HealBot_Globals.MacroSuppressSound==1 and sysSoundSFX=="1" then smName=smName.."/console Sound_EnableSFX 0;\n" end
+        if HealBot_Globals.MacroSuppressSound==1 and sysSoundSFX=="1" then smName=smName.."/console Sound_EnableSFX 0\n" end
         if spellTar then smName=smName.."/target "..HealBot_GetUnitName(unit)..";\n" end
         if spellTrin1 then smName=smName.."/use 13;\n" end
         if spellTrin2 then smName=smName.."/use 14;\n" end
         if HealBot_Config.MacroUse10==1 then smName=smName.."/use 10;\n" end
-        if HealBot_Globals.MacroSuppressSound==1 and sysSoundSFX=="1" then smName=smName.."/console Sound_EnableSFX 1;\n" end
+        if HealBot_Globals.MacroSuppressSound==1 and sysSoundSFX=="1" then smName=smName.."/console Sound_EnableSFX 1\n" end
         smName=smName..scText
         if spellAvoidBC then smName=smName.."/use 4;" end
         if strlen(smName)>255 then`
```

**Comment by vampik** [comment]
I wanted to post this in the bug reports section of the HealBot forum, but the posting on the forum is impossible. I get "Error Incorrect captcha. Please try again." when I try to post on the forum, but there is no captcha, only the following message "Invalid site key. Please see the FAQ."

**Comment by davidmarshall15** [comment]
ok I have removed the captcha for now, thanks for bringing this to my attention and thanks for the fix!

**Comment by vampik** [comment]
The changelog says you fixed this bug in 6.2.2.0, but this is not true, the file is unchanged.
Furthermore, you changed Sound_EnableSFX to Sound_EnableErrorSpeech but didn't apply the fix.
